src/lib/structures/CommandStore.ts

```diff
@@ -64,6 +70,12 @@ export class CommandStore extends AliasStore<Command> {
 			}
 		}
 
+		// If the default behavior is set to bulk overwrite, handle it as such and return.
+		if (defaultBehaviorWhenNotIdentical === RegisterBehavior.BulkOverwrite) {
```

Comment:
let's be consistent and use getDefaultBehaviorWhenNotIdentical() instead

Reply:
oh i missed that exists. Will change it after I look into the args unwrap issue.

Reply:
Resolved